src/app.js

```diff
@@ -322,11 +327,11 @@ class Messenger extends EventEmitter {
   //////////
 
   getCacheKey(senderId/*: number */)/*: string */ {
-    return `${config.get('facebook.appId')}-${senderId}`;
```

Comment:
This originally came from the quizbot repo. But it turns out the redis cache backend already has namespacing prefixes built in already. Making this shorter makes it much easier to browse the redis cache for debugging